app/connectors/IFSBusinessDetailsConnector.scala

```diff
@@ -67,4 +94,39 @@ class IFSBusinessDetailsConnectorImpl @Inject() (http: HttpClient, appConfig: Ap
 
     EitherT(get[Api1871Response](http, context))
   }
+
+  def createBroughtForwardLoss(
+      data: CreateBroughtForwardLossRequestData)(implicit hc: HeaderCarrier, ec: ExecutionContext): ApiResultT[api_1500.SuccessResponseSchema] = {
+    val url     = createBroughtForwardLossUrl(data.taxableEntityId)
+    val context = appConfig.mkMetadata(IFSApiName.Api1500, url)
+
+    EitherT(post[CreateBroughtForwardLossRequestBody, Api1500Response](http, context, data.body))
+  }
+
+  def updateBroughtForwardLoss(
+      data: UpdateBroughtForwardLossRequestData)(implicit hc: HeaderCarrier, ec: ExecutionContext): ApiResultT[api_1501.SuccessResponseSchema] = {
+    val url     = updateBroughtForwardLossUrl(data.taxableEntityId, data.lossId)
+    val context = appConfig.mkMetadata(IFSApiName.Api1501, url)
+
+    EitherT(put[UpdateBroughtForwardLossRequestBody, Api1501Response](http, context, data.body))
+  }
+
+  def getBroughtForwardLoss(taxableEntityId: Nino, lossId: BusinessId)(implicit
+      hc: HeaderCarrier,
+      ec: ExecutionContext): ApiResultT[api_1502.SuccessResponseSchema] = {
+    val url     = retrieveBroughtForwardLossUrl(taxableEntityId, lossId)
```

Comment:
In your next PR please can you change this to getBroughtForwardLossUrl to be consistent with the others